lib/tasks/data/export_workspace_data.rake

```diff
@@ -0,0 +1,92 @@
+# WORKSPACE_DATA_S3_BUCKET='s3-bucket-name' STATUSES_TO_EXPORT='False,Verified' bundle exec rake check:data:export_workspace_data[workspace-slug-1,workspace-slug-2,...,workspace-slug-N]
+
+namespace :check do
+  namespace :data do
+    desc 'Export all workspace data into a CSV uploaded to S3'
+    task export_workspace_data: :environment do |_t, params|
+      old_logger = ActiveRecord::Base.logger
+      ActiveRecord::Base.logger = nil
+
+      statuses_to_export = ENV.fetch('STATUSES_TO_EXPORT').to_s.split(',')
+      bucket_name = ENV.fetch('WORKSPACE_DATA_S3_BUCKET')
+      region = 'eu-west-1'
+      begin
+        s3_client = Aws::S3::Client.new(region: region)
+      rescue Aws::Sigv4::Errors::MissingCredentialsError
+        puts 'Please provide the AWS credentials.'
+        exit 1
+      end
+
+      def object_uploaded?(s3_client, bucket_name, object_key, file_path)
+        response = s3_client.put_object(
+          bucket: bucket_name,
+          key: object_key,
+          body: File.read(file_path)
+        )
+        if response.etag
+          return true
+        else
+          return false
+        end
+      rescue StandardError => e
+        puts "Error uploading S3 object: #{e.message}"
+        return false
+      end
+
+      slugs = params.to_a
+      slugs.each do |slug|
```

Comment:
I suggest to query teams here i.e. Team.where(slug: slugs).find_each do |team|

Reply:
Thanks, changed!